jobs/v4/howto/basic_client_event.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package howto
+
+import (
+	"context"
+	"fmt"
+	"io"
+  "time"
```

Comment:
Need to run `gofmt` - this is why the Go 1.12 Kokoro test failed.

Reply:
It fixed some formatting errors. But now Go 1.9/1.10/1.11 are not passing due to this issue:  
```
get "google.golang.org/api/jobs/v3": found meta tag get.metaImport{Prefix:"google.golang.org/api", VCS:"git", RepoRoot:"https://code.googlesource.com/google-api-go-client"} at https://google.golang.org/api/jobs/v3?go-get=1
get "google.golang.org/api/jobs/v3": verifying non-authoritative meta tag
package google.golang.org/api/jobs/v3: cannot find package "google.golang.org/api/jobs/v3" in any of:
	/usr/local/go/src/google.golang.org/api/jobs/v3 (from $GOROOT)
	/tmpfs/src/gopath/src/google.golang.org/api/jobs/v3 (from $GOPATH)
real	1m18.862s
user	0m6.875s
sys	0m1.794s
```
But Go 1.12 blazes through.